craco.config.cjs

```diff
@@ -29,25 +32,30 @@ module.exports = {
     },
   },
   webpack: {
-    plugins: [
-      new VanillaExtractPlugin({ identifiers: 'short' }),
-      new DefinePlugin({
-        'process.env.REACT_APP_GIT_COMMIT_HASH': JSON.stringify(commitHash.toString()),
-      }),
-    ],
+    plugins: [new VanillaExtractPlugin({ identifiers: 'short' })],
     configure: (webpackConfig) => {
-      const instanceOfMiniCssExtractPlugin = webpackConfig.plugins.find(
-        (plugin) => plugin instanceof MiniCssExtractPlugin
-      )
-      if (instanceOfMiniCssExtractPlugin !== undefined) instanceOfMiniCssExtractPlugin.options.ignoreOrder = true
+      // Extend process.env with dynamic values (eg commit hash).
+      Object.assign(webpackConfig.plugins.find((plugin) => plugin instanceof DefinePlugin).definitions['process.env'], {
+        REACT_APP_GIT_COMMIT_HASH: JSON.stringify(commitHash),
```

Comment:
why move this here? was it not working before?

Reply:
This was moved here to avoid adding another plugin to the stack, as it adds processing time. I assumed it was better to re-use the existing plugin, which is already installed by CRA.